.github/workflows/build-fdroid-apk.yml

```diff
@@ -0,0 +1,51 @@
+name: Build F-Droid apk
+
+on:
+  push:
+    branches:
+      - develop
+    paths-ignore:
+      - '**.md'
+      - '.github/**'
+  pull_request:
+    paths:
+      - 'subscriptions/subscriptions-api/**'
+      - 'subscriptions/subscriptions-dummy-impl/**'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_fdroid_apk:
+    name: Build F-Droid apk
+    runs-on: android-large-runner
```

Comment:
this is a relatively small job, why do you want to use a paid runner instead of a free one?

Reply:
Relative to other CI checks that we trigger on PRs, it is very similar, which is why I used the same runner. Isn't the `ubuntu-latest` runner free only for X minutes / month, and then it becomes paid anyway? Regardless, I'm ok with running this on `ubuntu-latest` given it won't trigger on most PRs. Just keep in mind that it will take 12-13 minutes to complete instead of 7-8.